modules/gitops-release-notes-1-7-0.adoc

```diff
@@ -0,0 +1,57 @@
+
+* With this update, to use the *Environments* feature on the *Developer* tab you will be forced to upgrade if you are on {gitops-title} lower than 1.7 and {product-title} higher than or on 4.15. link:https://issues.redhat.com/browse/GITOPS-2415[GITOPS-2415] 
+
+* With this update, applications can be created in any namespace in the same cluster and still managed by the same control-plane’s ArgoCD instance by adding a new label `argocd.argoproj.io/managed-by-cluster-argocd` to the namespace added in `spec.sourceNamespaces` of the Argo CD custom resource. link:https://issues.redhat.com/browse/GITOPS-2341[GITOPS-2341]
+
+:FeatureName: Argo CD Applications controller
+include::snippets/technology-preview.adoc[]
+
+
+[id="fixed-issues-1-7-0_{context}"]
+== Fixed issues
+
+The following issues have been resolved in the current release:
+
+* From {product-title} 4.12, the console installation will be optional. This fix updates {gitops-title} Operator to prevent problems with the operator if the console is not installed.  link:https://issues.redhat.com/browse/GITOPS-2292[GITOPS-2292]
+
+* Before this update, {gitops-title} releases were affected by an issue with dex pods failing with `CreateContainerConfigError` when a `SCC` of `anyuid` is assigned to the dex `serviceAccount`. This update fixes this issue by assigning a default user id to the dex container. link:https://issues.redhat.com/browse/GITOPS-2235[GITOPS-2235]
+
+* Before this update, {gitops-title} can make use of RHSSO (KeyCloak) through OIDC in addition to Dex. However, with a recent security fix applied, the certificate of RHSSO cannot be validated when it is setup with a certificate which is not signed by one of the well known certificate authorities. This update, fixes this issue and you can provide a custom certificate which will be used in verifying the Keycloak's TLS certificate when communicating with it. You can add the `rootCA` to their Argo CD custom resource `.spec.keycloak.rootCA` field. The operator reconciles to this change and updates the `oidc.config` in `argocd-cm` configmap with the PEM encoded root certificate. link:https://issues.redhat.com/browse/GITOPS-2214[GITOPS-2214]
+
+* Before this update, the application controllers were restarted multiple times in a large set of applications caused due to unresponsiveness of liveness probes. This update fixes this issue by removing the liveness probe in the application controller `statefulset`. link:https://issues.redhat.com/browse/GITOPS-2153[GITOPS-2153]
+
+[id="known-issues-1-7-0_{context}"]
+== Known issues
+
+* Before this update, operator was not reconciling **mountsatoken** and ServiceAccount settings for the repository server. While this has been fixed, deletion of ServiceAccount does not revert to default. link:https://issues.redhat.com/browse/GITOPS-1873[GITOPS-1873]
```

Comment:
Suggested version:

* Before this update, the Operator did not reconcile the **mountsatoken** and `ServiceAccount` settings for the repository server. While this has been fixed, deletion of the service account does not revert to default.
+
Workaround: Manually set the ``spec.repo.serviceaccount` field to the `default` service account.